jdaviz/configs/specviz/plugins/line_analysis/line_analysis.py

```diff
@@ -332,7 +332,44 @@ def _uncertainty(result):
         for function in FUNCTIONS:
             # TODO: update specutils to allow ALL analysis to take regions and continuum so we
             # don't need these if statements
-            if function == "Equivalent Width":
+            if function == "Line Flux":
+                flux_unit = spec_subtracted.flux.unit
+                # If the flux unit is equivalent to Jy, or Jy per spaxel for Cubeviz,
+                # enforce integration in frequency space
+                if (flux_unit.is_equivalent(u.Jy) or
+                        flux_unit.is_equivalent(u.Jy/u.sr)):
+                    # Perform integration in frequency space
+                    freq_spec = Spectrum1D(
+                        spectral_axis=spec_subtracted.spectral_axis.to(u.Hz,
+                                                                       equivalencies=u.spectral()),
+                        flux=spec_subtracted.flux)
```

Comment:
This is dropping the uncertainties (which will also need to have their units converted), resulting in no uncertainties on the returned line flux.